### DISCUSSION TITLE
Ascender accents, centered over letter or stem?

### DISCUSSION THREAD

**Discussion by weiweihuanghuang** [koeberlin/Latin-Character-Sets#13] [General]
I've seen discussion about this somewhere before, maybe it was Twitter or an obscure GitHub issue but I can't find it again so I thought it might be good to put it here too:

Should the accents over ascender letters such as h, k, b, d for letters such as ĥ ȟ ḧ ḣ ḱ ǩ ḃ ḋ be centered over the letter or the stems? Is it a case of differnet languages having different requirements? Most fonts I see it largely centered over the letter however San Francisco centers it over the stem.

**Comment by koeberlin** [comment]
Here’s the discussion: [https://mastodon.green/@eWalthert@typo.social/109907977429861949](https://mastodon.green/@eWalthert@typo.social/109907977429861949)

**Comment by weiweihuanghuang** [comment]
I found an old Twitter discussion too:
https://twitter.com/kaja_slojewska/status/1328490359630950402

**Comment by colinmford** [comment]
There's no good solution here, but I think above the stem presents fewer problems for corners of the design space that have heavier weights/narrower widths/wider accents
